src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/events/handlers/MigrateVideoBookingEventHandler.kt

```diff
@@ -30,12 +33,17 @@ class MigrateVideoBookingEventHandler(
         migrateVideoBookingService.migrate(bookingToMigrate)
       }
         .onFailure {
-          // TODO temporary telemetry details
-          telemetryService.capture("Failed migration of booking ${event.additionalInformation.videoBookingId} from whereabouts-api")
+          telemetryService.track(MigratedBookingFailureTelemetryEvent(event.additionalInformation.videoBookingId))
```

Comment:
_Note: the exception is not being logged here ... should it be?_

Reply:
Is the onFailure() catching the exception? Just wondering if the message will go onto the DLQ?

Reply:
It gets re-thrown so yes it will (still) go onto the DLQ, the `onFailure` is here so we can record it as telemetry.